server/utils/userUtils.ts

```diff
@@ -1,6 +1,16 @@
 import { Request } from 'express'
-import { ProbationRegion } from '../@types/shared'
+import { ProbationRegion, TemporaryAccommodationUser as User } from '../@types/shared'
+import disabledRegions from './disabledRegions'
+import config from '../config'
 
-export default function filterProbationRegions(regions: ProbationRegion[], request: Request) {
+export function filterProbationRegions(regions: ProbationRegion[], request: Request) {
   return [regions.find(region => region.id === request.session.probationRegion.id)]
 }
+
+export function hasRegionEnabled(user: User): boolean {
```

Comment:
Similar comment to above on naming. Maybe `isApplyEnabledForUser`?

Reply:
:recycle: